.pre-commit-config.yaml

```diff
@@ -1,52 +1,54 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
```

Comment:
Can you please remove the changes to this file from the PR?

Reply:
There was no option to comment on this til now.... but I fixed a few weeks ago.